### PR TITLE
Clarifying README

### PR DESCRIPTION
**Pull request by slodki**
Clarifying product name and its licensing.
Related to #60, #1230, #2982, #3029, #5342, #17996.
Based on @chrisdias [comment](https://github.com/Microsoft/vscode/pull/2983#issuecomment-183547642) and [comment](https://github.com/Microsoft/vscode/issues/60#issuecomment-161792005)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
 
-VS Code is updated monthly with new features and bug fixes. You can download it for Windows, Mac and Linux on [VS Code's website](https://code.visualstudio.com/Download). To get the latest releases everyday, you can install the [Insiders version of VS Code](https://code.visualstudio.com/insiders). This builds from the master branch and is updated at least daily.
-
 <p align="center">
   <img alt="VS Code in action" src="https://cloud.githubusercontent.com/assets/11839736/16642200/6624dde0-43bd-11e6-8595-c81885ba0dc2.png">
 </p>
 
+Code is updated monthly with new features and bug fixes.
+You can download sources of released versions from [GitHub Releases](https://github.com/Microsoft/vscode/releases).
+Building binaries for Windows, OS X and Linux [instructions](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#build-and-run-from-source)
```
Comment (torn4dom4n) [inline README.md]: Using Mac instead of OS X?

Comment (mueller-ma) [inline README.md]: Anyway use macOS. OS X is outdated: https://en.wikipedia.org/wiki/MacOS

Comment (slodki) [inline README.md]: I've just cited page at given URL - I don't know if these instructions work for the lastest macOS version.

File: README.md [hunk 2]
```diff
@@ -41,6 +44,20 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
+## Related Products
+Microsoft is distributing its own build as Microsoft Visual Studio Code. 
+You can download packages for Windows, Mac and Linux on [VS Code's website](https://code.visualstudio.com/Download). 
+
+It's based on Code Open Source with some changes:
+* Microsoft branding
+* licensed under Microsoft [closed source custom product license](https://code.visualstudio.com/License) (EULA)
+* Extension Marketplace supported
+* crash reporting and telemetry enabled
+* user configuration and extensions stored in different directory then Code - Open Source
```
Comment (dougireton) [inline README.md]: Typo: s/then/than/

File: README.md [hunk 3]
```diff
@@ -41,6 +44,20 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
 
+## Related Products
+Microsoft is distributing its own build as Microsoft Visual Studio Code. 
+You can download packages for Windows, Mac and Linux on [VS Code's website](https://code.visualstudio.com/Download). 
+
+It's based on Code Open Source with some changes:
+* Microsoft branding
+* licensed under Microsoft [closed source custom product license](https://code.visualstudio.com/License) (EULA)
+* Extension Marketplace supported
+* crash reporting and telemetry enabled
```
Comment (jens1o) [inline README.md]: maybe add that both can be disabled?

**Comment by chrisdias** [comment]
I've made a few changes to the `readme` inspired by this PR, so I'm going to close this. We are also writing a doc that makes it clear what is in the Visual Studio Code distro vs. the repository. Once that doc is published (probably on the wiki) I'll update the readme again with a link.

**Pull request closed without merging**